src/Files.App/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -603,6 +604,14 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseLayoutMenuItems(Curren
                     ShowInRecycleBin = true,
                 },
                 new ContextMenuFlyoutItemViewModel()
+                {
+                    Text = "RestoreRecycleBin".GetLocalizedResource(),
```

Comment:
Per the requirements this should be "RestoreAllItems"

Reply:
On it.